src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/AssemblyQualifiedNamespaceSymbol.cs

```diff
@@ -18,14 +18,20 @@ internal sealed class AssemblyQualifiedNamespaceSymbol : ParentSymbol, ITypeOrNa
         // AssemblyQualifiedNamespaceSymbol
         // ----------------------------------------------------------------------------
 
-        public bool IsType()
+        public bool IsType
```

Comment:
nit: we can also use the Syntax:
public bool IsType => false;
for concise representation.